### PR TITLE
chore: bump alpine to 3.15

### PR DESCRIPTION
**Pull request by faiq**
**What problem does this PR solve?**:
Adds more accepted algorithms so we can bump alpine which brings in open-ssh 8.8 

**Which issue(s) does this PR fix?**:

* https://jira.d2iq.com/browse/D2IQ-NUMBER

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by dkoshkin** [comment]
Any concerns with this change for existing users? Do the need to change their existingSSH keys?

File: pkg/packer/manifests/aws/packer.json.tmpl [hunk 1]
```diff
@@ -73,6 +73,7 @@
       "iam_instance_profile": "{{user `iam_instance_profile`}}",
       "skip_profile_validation": "{{user `skip_profile_validation`}}",
       "ssh_username": "{{user `ssh_username`}}",
+      "ssh_key_exchange_algorithms": ["curve25519-sha256@libssh.org", "ecdh-sha2-nistp256", "ecdh-sha2-nistp384", "ecdh-sha2-nistp521", "diffie-hellman-group14-sha1","diffie-hellman-group1-sha1"],
```
Comment (jkoelker) [inline pkg/packer/manifests/aws/packer.json.tmpl:76]: I think we need to replicate this in:

https://github.com/mesosphere/konvoy-image-builder/blob/main/pkg/packer/manifests/vsphere/packer.json.tmpl
https://github.com/mesosphere/konvoy-image-builder/blob/main/pkg/packer/manifests/vsphere/base/packer-base.json

To make sure that `diffie-hellman-group1-sha1` is an available algo there too

File: pkg/packer/manifests/aws/packer.json.tmpl [hunk 2]
```diff
@@ -185,7 +186,7 @@
       "playbook_file": "./ansible/provision.yaml",
       "user": "{{user `ssh_username`}}",
       "ansible_env_vars": [
-        "ANSIBLE_SSH_ARGS='{{user `existing_ansible_ssh_args`}} -o IdentitiesOnly=yes'",
+        "ANSIBLE_SSH_ARGS='{{user `existing_ansible_ssh_args`}} -o IdentitiesOnly=yes -o HostkeyAlgorithms=+ssh-rsa'",
```
Comment (jkoelker) [inline pkg/packer/manifests/aws/packer.json.tmpl:189]: I think we need to add this to https://github.com/mesosphere/konvoy-image-builder/blob/main/pkg/packer/manifests/vsphere/packer.json.tmpl as well.

**Review by jkoelker — COMMENTED** [review]
LGTM, Just a note to add it to the other provider where approps.

**Pull request merged**